src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+    />
+    <title>Wbpack Exercise</title>
+  </head>
+  <body>
+    <!-- <div id="todo-list"></div> -->
+
+      <div id="todo-list" class="todo-list"></div>
+    </div>
+
```

Comment:
- [ ] I can see that you have done well to implement and generate your HTML dynamically with Javascript, but on line 16 of your code you have a div that is not opened at any point, I suggest you remove it as it corresponds with HTML best practices
- [ ] Please remove all commented code to keep your codes clean

Reply:
Okay Noted. Thanks